Assets/Scripts/BaseCode/Entity.cs

```diff
@@ -10,7 +10,7 @@ public class Entity : MonoBehaviour
     [SerializeField]
     public float health;
 
-    [EasyButtons.Button]
+    //[EasyButtons.Button]
```

Comment:
Hey if possible you shouldn't change other people's code, this can cause a lot of problems if you change something and someone else made the same change.

Reply:
Sorry, it was the only way I could implement cinemachine on my end before I updated Unity.